inst/examples/dashboard/header.R

```diff
@@ -41,23 +41,112 @@ farItems <- list(
     ariaLabel = "Grid view",
     iconOnly = TRUE,
     iconProps = list(iconName = "Tiles")
-  ),
-  list(
-    key = "info",
-    id = "aboutInfo",
-    href = "#!/about",
-    text = "Info",
-    ariaLabel = "Info",
-    iconOnly = TRUE,
-    iconProps = list(iconName = "Info")
   )
 )
 
-command_bar <- CommandBar(items = items,
+command_bar <- CommandBar(
+  items = items,
   farItems = farItems,
-  style = list(width = "100%"))
+  style = list(width = "100%")
+)
+
+
+appsilon_header <- function() {
+
+  appsilon_breakpoints <- breakpointSystem(
+    "appsilon-breakpoints",
+    breakpoint("xs", min = 320),
+    breakpoint("s", min = 428),
+    breakpoint("m", min = 728),
+    breakpoint("l", min = 1024),
+    breakpoint("xl", min = 1200)
+  )
+
+  gridPanel(
+    id = "app_header",
+
+    class = "mobile-collapsed",
+
+    breakpoint_system = appsilon_breakpoints,
+
+    areas = list(
+      default = c(
+        "logo . info mobile_controls",
+        "separator separator separator separator",
+        "title title title title",
+        "menu menu menu menu",
+        "cta cta cta cta"
+      ),
+      l = "logo separator title mobile_controls . menu info cta"
+    ),
 
-logo <- img(src = "appsilon-logo.png", class = "logo")
-title <- div(Text(variant = "xLarge", "shiny.fluent"), class = "title")
+    columns = list(
+      default = "auto 1fr auto auto",
+      l = "auto 1px auto auto 1fr auto auto auto"
+    ),
 
-header <- tagList(logo, title, command_bar)
+    rows = list(
+      default = "auto auto auto auto auto",
+      l = "40px"
+    ),
+
+    gap = list(
+      default = "0px",
+      l = "16px"
+    ),
+
+    logo = img(src = "appsilon-logo.png", class = "logo"),
+
+    separator = div(class = "app_header_vertical_separator mobile-toggled"),
+
+    title = div(
+      "shiny.fluent",
+      class = "app_header_title mobile-toggled"
+    ),
+
+    commandBar = command_bar,
```

Comment:
It seems that `commandBar` area is not defined and thus it doesn't appear in the app at all. That's fine as it wasn't doing anything anyway (just some buttons without any function), but let's remove the dead code (including the `command_bar` definition above).

Reply:
While this assignment can be removed in favour of only writing command_bar, I believe I shouldn't be removing the command_bar definition since it contains in itself the buttons for download, upload etc.